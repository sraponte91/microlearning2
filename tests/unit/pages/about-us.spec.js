import { config, createLocalVue, mount } from '@vue/test-utils'
import VueMeta from 'vue-meta'
import LaFundacion from '@/pages/about-us.vue'
// create vue with vue-meta
const localVue = createLocalVue()
localVue.use(VueMeta, { keyName: 'head' })

const mock = {
    localVue,
    mocks: {
        $route: {
            path: '/some/path'
        }
    }
}

const wrapper = mount(LaFundacion, mock)

describe('about-us.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
    test('has correct <head> content', () => {
        expect(wrapper.vm.$metaInfo.title).toBe(config.mocks['$constants'].aboutUs.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'title'
        ).content).toBe(config.mocks['$constants'].aboutUs.meta.metaTitle)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'description'
        ).content).toBe(config.mocks['$constants'].aboutUs.meta.metaDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:title'
        ).content).toBe(config.mocks['$constants'].aboutUs.meta.ogTitle)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:description'
        ).content).toBe(config.mocks['$constants'].aboutUs.meta.ogDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:image'
        ).content).toBe(config.mocks['$constants'].baseURL + config.mocks['$constants'].aboutUs.meta.ogImage)
    })
})
