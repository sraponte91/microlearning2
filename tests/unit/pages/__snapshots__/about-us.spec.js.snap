// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about-us.vue Renders correctly 1`] = `
<main>
  <section
    class="full-image"
  >
    <div
      class="position-absolute"
    >
      <img
        alt="About Us"
        class="object-fit-cover"
        src="/images/seccion-about/portada.jpg"
      />
    </div>
  </section>
   
  <section
    class="about-content"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2 text-center"
        >
          <p
            class="ff-nunito caption"
          >
            Who we are
          </p>
           
          <h1
            class="ff-sans-b"
          >
            About Us
          </h1>
           
          <p>
            
            More than 75 years of experience and support from the Bolívar
            Business Group has enabled our foundation to expand its horizons
            and to provide the trust that has distinguished our organization
            throughout Colombia with our wide array of programs.
          
          </p>
           
          <p>
            
            Our mission is to support and promote life-changing and
            high-impact projects that develop skills in individuals,
            communities and organizations in order to build a more just,
            innovative and equitable society.
          
          </p>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="about-lists"
  >
    <div
      class="container"
    >
      <div
        class="d-flex multiple-reapt align-items-start flex-wrap-991"
      >
        <div
          class="txt-points position-relative"
        >
          <div
            class="pra-p"
          >
            <h3
              class="ff-sans-b"
            >
              Aflora
            </h3>
             
            <p>
              
              Our goal is to work with Colombian companies exhibiting a high
              potential for social impact in order to strengthen their
              management and growth through professional training and
              guidance. These companies work with a network of successful
              business professionals from around the country, as well as
              senior management representatives from the Bolívar Business
              Group, who share their knowledge in order to build
              entrepreneurial initiatives.
            
            </p>
             
            <p>
              
              Thanks to Emprende País, a program of the Fundación Bolívar
              Davivienda, we have supported and strengthened more than 213
              companies that have worked hand-in-hand with our program to
              achieve their goals, creating high-impact in their industries in
              a sustainable and efficient way.
            
            </p>
             
            <a
              class="btn btn-primary"
              to="/aflora"
            >
              
              Learn More
            
            </a>
          </div>
        </div>
         
        <div
          class="img-points"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0006_aflora.jpg"
          />
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start part-two flex-wrap-991"
      >
        <div
          class="img-points order-2-991"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0003_emprende_pais.jpg"
          />
        </div>
         
        <div
          class="two-div-in-one position-relative order-1-991"
        >
          <div
            class="txt-points position-relative"
          >
            <div
              class="pra-p"
            >
              <h3
                class="ff-sans-b"
              >
                Emprende País
              </h3>
               
              <p>
                
                Our goal is to work with Colombian companies exhibiting a high
                potential for social impact in order to strengthen their
                management and growth through professional training and
                guidance. These companies work with a network of successful
                business professionals from around the country, as well as
                senior management representatives from the Bolívar Business
                Group, who share their knowledge in order to build
                entrepreneurial initiatives.
              
              </p>
               
              <p>
                
                Thanks to Emprende País, a program of the Fundación Bolívar
                Davivienda, we have supported and strengthened more than 213
                companies that have worked hand-in-hand with our program to
                achieve their goals, creating high-impact in their industries
                in a sustainable and efficient way.
              
              </p>
               
              <a
                class="btn btn-primary"
                to="/emprende-pais"
              >
                
                Learn More
              
              </a>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start flex-wrap-991"
      >
        <div
          class="txt-points position-relative"
        >
          <div
            class="pra-p"
          >
            <h3
              class="ff-sans-b"
            >
              Filarmónica Joven de Colombia
            </h3>
             
            <p>
              
              Over 100 musicians form this exemplary orchestra, which has been
              recognized for its talent and world-class performances through
              its array of professional shows on different musical stages
              around the world.
            
            </p>
             
            <p>
              
              Created in 2010, the Filarmónica Joven de Colombia has launched
              the musical careers of more than 400 young musicians who have
              achieved their dreams of showcasing their talent by being part
              of one of the best symphonic youth orchestras. The pride that
              their country has for this orchestra motivates these young
              musicians to continue forward with their careers.
            
            </p>
             
            <p>
              
              The Filarmónica Joven de Colombia partners with multiple
              national and international institutes. Thanks to its partnership
              with the YOA (Orchestra of America), the orchestra maintains a
              strong bond with teachers from the University of Minnesota, the
              University of Kansas, Ball State University, the Minnesota
              Orchestra, and the New York Metropolitan Orchestra. Since 2014,
              the orchestra has developed important relationships with the
              Houston Symphony Orchestra, the Mahler Chamber Orchestra, the
              Vienna Philharmonic Orchestra, and the Escuela Superior de
              Música Reina Sofía in Madrid (with whom the orchestra
              participates in an exchange program agreement).
            
            </p>
             
            <a
              class="btn btn-primary"
              to="/filarmonica-joven-de-colombia"
            >
              Learn More
            </a>
          </div>
        </div>
         
        <div
          class="img-points"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0002_filarmonica.jpg"
          />
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start part-two flex-wrap-991"
      >
        <div
          class="img-points order-2-991"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0000_voluntariado.jpg"
          />
        </div>
         
        <div
          class="two-div-in-one position-relative order-1-991"
        >
          <div
            class="txt-points position-relative"
          >
            <div
              class="pra-p"
            >
              <h3
                class="ff-sans-b"
              >
                Voluntariado Familia Bolívar
              </h3>
               
              <p>
                
                Being a part of the Bolívar Business Group means being part of
                a family that is committed to the development of the regions
                in which our offices are located.
              
              </p>
               
              <p>
                
                Our volunteers are both employees and partners who are
                committed and willing to work and contribute to the social
                sector of Colombia. Our 5,000 volunteers have impacted more
                than 70,000 people with our Great Bolívar Family Volunteering
                Movement. We are proud to encourage our partners to contribute
                to the social sector both inside and outside the country.
              
              </p>
               
              <a
                class="btn btn-primary"
                to="/voluntariado-fundacion-bolivar"
              >
                Learn More
              </a>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start flex-wrap-991"
      >
        <div
          class="txt-points position-relative"
        >
          <div
            class="pra-p"
          >
            <h3
              class="ff-sans-b"
            >
              Cultivarte
            </h3>
             
            <p>
              
              This program is supported by Fundación Bolívar Davivienda. It
              offers playful learning spaces where beneficiaries are able to
              enhance their talents and inspire others to achieve their goals
              by developing their skills.
            
            </p>
             
            <p>
              
              In Cultivarte, we focus on five areas to develop our plan. These
              areas stimulate competencies and enable beneficiaries to develop
              in a holistic manner with their environment. This way we rewrite
              history, not only of children and adolescents, but also of their
              communities.
            
            </p>
             
            <a
              class="btn btn-primary"
              to="/cultivarte"
            >
              
              Learn More
            
            </a>
          </div>
        </div>
         
        <div
          class="img-points"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0004_cultivarte.jpg"
          />
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start part-two flex-wrap-991"
      >
        <div
          class="img-points order-2-991"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0001_inversion social.jpg"
          />
        </div>
         
        <div
          class="two-div-in-one position-relative order-1-991"
        >
          <div
            class="txt-points position-relative"
          >
            <div
              class="pra-p"
            >
              <h3
                class="ff-sans-b"
              >
                Inversión Social
              </h3>
               
              <p>
                
                Our belief in transformative ideas has led us to support
                projects that build skills, empower communities and promote
                sustainability.
              
              </p>
               
              <p>
                
                In order to bring life to these projects and encourage them on
                their path to self-improvement, we request project proposals
                from various regions of Colombia, evaluate potential projects
                and, eventually, co-finance them thanks to the Fundación
                Bolívar Davivienda – Inversión Social program.
              
              </p>
               
              <p>
                
                We support organizations that strive to protect children and
                adolescents and prevent teenage pregnancy, among other social
                issues, in order to achieve transformative results.
              
              </p>
               
              <a
                class="btn btn-primary"
                to="/inversion-social"
              >
                
                Learn More
              
              </a>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="d-flex multiple-reapt align-items-start flex-wrap-991"
      >
        <div
          class="txt-points position-relative"
        >
          <div
            class="pra-p"
          >
            <h3
              class="ff-sans-b"
            >
              Becas al Talento
            </h3>
             
            <p>
              
              The education and formation of Colombian youth is very important
              to the Fundación Bolívar Davivienda. Our scholarship program
              aims to support outstanding, high-performing students who do not
              have the resources to pay for their university or to develop
              their talents.
            
            </p>
             
            <p>
              
              Many students have completed their studies thanks to our Becas
              (scholarships) program. We are proud to be part of the
              educational formation of so many students who will become
              skilled professionals prepared to create positive change
              and transform the face of Colombia.
            
            </p>
             
            <a
              class="btn btn-primary"
              to="/becas-al-talento"
            >
              
              Learn More
            
            </a>
          </div>
        </div>
         
        <div
          class="img-points"
        >
          <img
            alt=""
            class="object-fit-cover"
            src="/images/seccion-about/aboutusimg_0005_becas.jpg"
          />
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="about-download"
  >
    <div
      class="container"
    >
      <h3>
        Downloads
      </h3>
       
      <ul
        class="d-flex"
      >
        <li>
          <a
            href="pdf/About_Us_Privacy_Policy_of_La_Fundacion.pdf"
          >
            Privacy Policy of La Fundación
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Privacy_Policy_of_Cultivarte.pdf"
          >
            Privacy Policy of Cultivarte
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Notice_of_Privacy_of_La_Fundacion.pdf"
          >
            Notice of Privacy of La Fundación
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Notice_of_Privacy_of_Cultivarte.pdf"
          >
            Notice of Privacy of Cultivarte
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Autorization_for_data protection_FBD_FCult.pdf"
          >
            Autorization for data protection
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Specific_regulations_per_project_FBD_FCult.pdf"
          >
            Specific regulations per project
          </a>
        </li>
         
        <li>
          <a
            href="pdf/About_Us_Terms_and_Conditions_FBD_Fcult.pdf"
          >
            Terms and Conditions
          </a>
        </li>
      </ul>
    </div>
  </section>
</main>
`;
